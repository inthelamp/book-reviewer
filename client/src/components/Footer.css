footer {
    margin-top: 5vh;
    text-align: right;    
}