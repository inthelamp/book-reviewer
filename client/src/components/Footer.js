import { Link } from 'react-router-dom'
import logo from '../assets/images/logo.png'

/**
 * A footer showing copy right
 */
const Footer = () => {
    return (
        <footer>
            <p><img className='App-logo' src={logo} alt='Logo' /> &nbsp;Copyright &copy; 2021, IntheLamp &nbsp;
            <Link to='/about'>About</Link>
            </p>
        </footer>
    )
}

export default Footer