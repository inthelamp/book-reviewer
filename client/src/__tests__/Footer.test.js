import React from 'react'
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import Footer from '../components/Footer'

it('Component: Footer rendering correctly', () => {  
    const tree = renderer
      .create(<Router><Footer /></Router>)
      .toJSON();
    expect(tree).toMatchSnapshot()
})
