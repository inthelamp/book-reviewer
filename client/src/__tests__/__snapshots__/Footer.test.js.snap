// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer rendering correctly 1`] = `
<footer>
  <p>
    <img
      alt="Logo"
      className="App-logo"
      src={Object {}}
    />
      Copyright © 2021, IntheLamp  
    <a
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
  </p>
</footer>
`;
